mobile/android/android-device-file-system.ts

```diff
@@ -14,8 +14,13 @@ export class AndroidDeviceFileSystem implements Mobile.IDeviceFileSystem {
 		private $injector: IInjector,
 		private $options: ICommonOptions) { }
 
-	public listFiles(devicePath: string): IFuture<void> {
-		return Future.fromResult();
+	public listFiles(devicePath: string, appIdentifier?: string): IFuture<any> {
+		let listCommandArgs = ["ls", "-al", devicePath];
```

Comment:
changing this to `-a` option only. This option **should** be available on all android devices